plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpClientOperationsOnInboundNextInterceptor.java

```diff
@@ -81,7 +82,7 @@ public void doInAfterTrace(SpanEventRecorder recorder, Object target, Object[] a
     }
 
     private boolean validate(final Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```

Comment:
I think `args.length < 2` would be better to change  `ArrayUtils.getLength(args) < 2`.